### PR TITLE
Issue #267 "day off" auto choosed cancel

### LINKED ISSUE #267: Day off game is selected when you edit game
1. Any tournament
2. Create game
3. Assign both teams to it
4. Save
5. Click edit

**AR:** Away team is set to DayOff

- [x] #278 Delete Day off selected automatically

- [ ] #277 Maunal Test

### PR DESCRIPTION
**Pull request by NataliiaBidiuk**
### Summary

When we want Edit schedule, day off  iD== HomeTeamId, so we added some conditions: don`t call function update when not day Off

closes #267 

### Areas Of Interest

_Any specific places you want to bring attention of the reviewers? Some places you are uncertain in/some design decisions. (If none remove this section)_

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js [hunk 1]
```diff
@@ -4,17 +4,21 @@ $(document).ready(function () {
...
+  var AwayTeamId = $(".away-team-select :selected").val();
 
   me.updateDropdownStates = function () {
     var dropDowns = [
       $(".round-select")[0],
       $(".home-team-select")[0],
       $(".away-team-select")[0]
     ];
-
+
```
Comment (volleymanagement-sonarqube) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js [hunk 2]
```diff
@@ -4,17 +4,21 @@ $(document).ready(function () {
...
       $(".round-select")[0],
       $(".home-team-select")[0],
       $(".away-team-select")[0]
     ];
-
+   
     dropDowns.forEach(function (dd) {
-      me.updateStateForDropDown(dd);
+      if (dd !== dropDowns[2] || HomeTeamId === AwayTeamId)
+        me.updateStateForDropDown(dd);
```
Comment (volleymanagement-sonarqube) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by volleymanagement-sonarqube** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js (hunk 1 above)
Comment (NataliiaBidiuk) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js]: it has a view without this line
image

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js [hunk 3]
```diff
@@ -13,8 +14,9 @@ $(document).ready(function () {
     ];
 
     dropDowns.forEach(function (dd) {
-      me.updateStateForDropDown(dd);
+        me.updateStateForDropDown(dd);
```
Comment (volleymanagement-sonarqube) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by volleymanagement-sonarqube** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**